scijava-common3/src/main/java/org/scijava/common3/Types.java

```diff
@@ -1,8 +1,8 @@
 /*
  * #%L
- * SciJava library for generic type reasoning.
```

Comment:
While I appreciate why you did this, it does feel strange to me that our type logic is now spread out into multiple repositories. I liked the simplicity of "If I need to reason about types then I always require `scijava-types`"...

With that being said, I'm not going to request changes, because like you said if you don't need the dependency it is nice.

Reply:
I agree it feels a little strange, but it's really the right place for it. What pushed me over the line was scijava-struct having a scijava-types and therefore guava dependency, solely so that it could call `Types.name`, `Types.raw`, and `Types.isAssignable`.